15/umbraco-cms/reference/security/two-factor-authentication.md

```diff
@@ -642,7 +652,7 @@ You should only customize the 2FA login screen in certain cases, for example:
 
 * If you have a provider that requires a non-numeric field or additional info.
 * If you have a provider that requires the user to scan a QR code, you should additionally show the QR code.
-* If you need to authenticate the user in a different way than the default [AuthenticationController](https://apidocs.umbraco.com/v14/csharp/api/Umbraco.Cms.Web.BackOffice.Controllers.AuthenticationController.html#Umbraco\_Cms\_Web\_BackOffice\_Controllers\_AuthenticationController\_PostVerify2FACode\_Verify2FACodeModel\_) in Umbraco.
+* If you need to authenticate the user in a different way than the default [AuthenticationController](https://apidocs.umbraco.com/v15/csharp/api/Umbraco.Cms.Web.BackOffice.Controllers.AuthenticationController.html#Umbraco_Cms_Web_BackOffice_Controllers_AuthenticationController_PostVerify2FACode_Verify2FACodeModel_) in Umbraco.
```

Comment:
This link doesn't seem to work? 🤔 
I also can't find the Umbraco.Cms.Web.Backoffice anywhere 😅 Might not be a thing anymore actually 🤔

Reply:
How about shortening this sentence instead:

```suggestion
* If you need to authenticate the user in a different way than the default option.
```